pkg/discord/main.go

```diff
@@ -60,6 +61,19 @@ func UserReact(s *discordgo.Session, m *discordgo.MessageReactionAdd) {
 		} else if err = s.GuildMemberRoleAdd(m.GuildID, m.UserID, global.NewRole); err != nil {
 			fmt.Println("error adding role,", err)
 		}
+	} else if channel, err := s.Channel(m.ChannelID); err == nil && channel.Type == discordgo.ChannelTypeGuildText && channel.ParentID == global.ProjectCategoryId && m.Emoji.Name == global.IssueEmoji {
+		if msg, err := s.ChannelMessage(m.ChannelID, m.MessageID); err != nil {
+			fmt.Println("error fetching message to create issue,", err)
+		} else {
```

Comment:
this can probably be simplified by adding a `return` instead of a `} else {` case ? lmk what you think or if i'm missing something.

Reply:
You're right, pushed up that change.

Reply:
thank you!